resilience4j-retry/src/main/java/io/github/resilience4j/retry/RetryConfig.java

```diff
@@ -150,7 +182,7 @@ public Builder(RetryConfig baseConfig) {
                 this.intervalFunction = (x) -> waitDuration.toMillis();
```

Comment:
Doesn't need to set `IntervalBiFunction`? 
```suggestion
                this.intervalBiFunction = (attempt, either) -> waitDuration.toMillis();
```

Reply:
done